### PR TITLE
Adds Chef Client release notes 12.19

### PR DESCRIPTION
**Pull request by kagarmoe**

### CODE REVIEW

File: chef_master/source/release_notes.rst [hunk 1]
```diff
@@ -5,6 +5,95 @@ Release Notes: chef-client 12.0 - 12.18
...
+* **Added ed25519 key support for for ssh connections.**
+
+Windows alternate user identity execute support
+-----------------------------------------------------
+
+The ``execute`` resource and similar resources such as ``script``, ``batch``, and ``powershell_script`` now support the specification of credentials on Windows so that the resulting process is created with the security identity that corresponds to those credentials.
+
+**Note**: When Chef is running as a service, this feature requires that the user that Chef runs as has 'SeAssignPrimaryTokenPrivilege' (aka 'SE_ASSIGNPRIMARYTOKEN_NAME') user right. By default only LocalSystem and NetworkService have this right when running as a service. This is necessary even if the user is an Administrator.
+
+This right bacn be added and checked in a recipe using this example:
```
Comment (davidwrede) [inline chef_master/source/release_notes.rst]: "bacn" looks like a typo to me. Please confirm the correct text.

**Comment by davidwrede** [comment]
Please make sure you update the title to reference 12.19.

File: chef_master/source/release_notes.rst (hunk 1 above)
Comment (kagarmoe) [reply chef_master/source/release_notes.rst]: "can"

File: chef_master/source/ohai.rst [hunk 2]
```diff
@@ -63,6 +63,8 @@ An automatic attribute is a specific detail about a node, such as an IP address,
 
 .. end_tag
 
+Changed in 8.23 to correctly detect Cumulus Linux platform and the GCC tool on Unix systems.
```
Comment (tas50) [inline chef_master/source/ohai.rst]: This isn't worth noting here

File: chef_master/source/ohai.rst [hunk 3]
```diff
@@ -338,6 +340,8 @@ The following list shows the type of plugins that are included with Ohai. See th
      platform.rb
      virtualization.rb
 
+New in 8.23, the sysconf plugin parses the output of the sysconf command. Changed in 8.23 to detect Windows/Linux/BSD guests running on the Veertu hypervisors, Windows guests running on Xen and Hyper-V hypervisors, and the EC2 plugin retrieves the AWS Account ID in addition to previous metadata.
```
Comment (tas50) [inline chef_master/source/ohai.rst]: Same with this one

**Review by tas50 — DISMISSED** [review]

File: chef_master/source/ohai.rst (hunk 2 above)
Comment (kagarmoe) [reply chef_master/source/ohai.rst]: OK

File: chef_master/source/ohai.rst (hunk 3 above)
Comment (kagarmoe) [reply chef_master/source/ohai.rst]: OK

**Comment by kagarmoe** [comment]
gif-keyboard-14009137962770216605

**Pull request merged**